### PR TITLE
Swift 5.7

### PR DESCRIPTION
**Pull request by finestructure**
- updates the build matrix to display Swift 5.7 and removes 5.3
- starts triggering 5.7
- stops triggering 5.3
- removes 5.3 builds

### CODE REVIEW

**Review by daveverwer — APPROVED** [review]

**Comment by daveverwer** [comment]
I'll sort out the Mac builder machines on Monday 👍

File: Resources/Markdown/docs/builds.md [hunk 1]
```diff
@@ -42,12 +42,11 @@ When running `xcodebuild`, we apply some heuristics to find the correct scheme t
 
 The build system aims to reproduce a real-world environment for building packages as much as possible. Ideally, if the Swift Package Index says it's compatible with Swift 5.1 and your project uses Swift 5.1, it should work. The most real-world way for us to build is to use multiple different versions of Xcode to process each package. We use the latest available version of Xcode that shipped with the release of Swift that we want to compile with as default.
 
-- Swift 5.1 builds with Xcode 11.3.1.
-- Swift 5.2 builds with Xcode 11.6.
 - Swift 5.3 builds with Xcode 12.4.
 - Swift 5.4 builds with Xcode 12.5.
 - Swift 5.5 builds with Xcode 13.1.
```
[the remark this answers is not in the document]
Comment (finestructure) [reply Resources/Markdown/docs/builds.md:47]: Actually, how about we just replace that list with

```
In order to find out which Xcode version a Swift version corresponds to, please refer to [swiftversion.net](https://swiftversion.net). We also show this as part of the build command on the details page of each individual build.
```

so we don't have to deal with updates?

[the remark this answers is not in the document]
Comment (daveverwer) [reply Resources/Markdown/docs/builds.md:47]: Yes this is good. No one is checking this to see the versions. We put the correct version on the build report.

**Comment by finestructure** [comment]
Here are the rollout steps:

- [ ] merge server PR [#2013](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/pull/2013)
- [ ] after auto-deploy to dev check 5.3 builds are deleted as expected
- [ ] tag release 2.77.0
- [ ] be ready to merge (i.e. deploy) builder changes in MR [!150](https://gitlab.com/finestructure/swiftpackageindex-builder/-/merge_requests/150) when the new release is deployed
	- since we're removing 5.3 at the same time, we need to align the deployment this time around
- [ ] remove 5.3 tag on builders

**Pull request merged**